services/jupyter/src/app/config.py

```diff
@@ -0,0 +1,30 @@
+import enum
+import os
+import secrets
+from typing import Optional
+
+from pydantic import AnyHttpUrl, BaseSettings, HttpUrl, validator
+
+
+class SettingsModeEnum(str, enum.Enum):
+    DEV = "dev"
+    PROD = "prod"
+    TEST = "test"
```

Comment:
It can be used in the CLI and at the container level, leaving it as a proposed interface.

Reply:
I am tempted to go in the direction of "everything nice to have (but that is not used) should go since we have less than a week left and we need to tidy up the code base"

Reply:
It's a nice thing to have indeed. Shall we mark this as resolved then?